db/migrate/20230117043156_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
```

Comment:
- [ ] Also here, the associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).  To show that, the  `'author_id'` reference the User class. Kindly write the query like this so the foreign key name will be `author_id`:


```ruby
add_reference :likes, :author, null: false, foreign_key: { to_table: :users }
```